system/web/routing/Router.cfc

```diff
@@ -631,7 +631,7 @@ component accessors="true" extends="coldbox.system.FrameworkSupertype" threadsaf
 
 	        // update, delete and show routes
 	        actionSet = filterRouteActions(
-	        	{ PUT = "update", PATCH = "update", POST = "create", DELETE = "delete", GET = "show" },
```

Comment:
Why did you remove `create`, this will break all resources that use a `POST` for creation though?

Reply:
You don’t `POST` to the `/:id` route. You post to the base route.

Reply:
That `POST` was a duplicate.

Reply:
Really, where is it defined, can't see it?

Reply:
Further down: https://github.com/ColdBox/coldbox-platform/blob/f3c873dec5169238b3653a1dcd4bb9d4298588c0/system/web/routing/Router.cfc#L649